src/main/resources/checkstyle-config.xml

```diff
@@ -30,7 +30,7 @@
     <module name="UnusedImports"/>
     <module name="IllegalImport">
       <!-- For "org.apache.commons.lang.*" classes use the corresponding class from "org.apache.commons.lang3.*" -->
-      <property name="illegalPkgs" value="com.google.hive12,com.beust.jcommander.internal,jersey.repackaged,org.apache.commons.lang"/>
+      <property name="illegalPkgs" value="com.google.hive12,jersey.repackaged,org.apache.commons.lang"/>
```

Comment:
Why was this changed? Rebase error? Maybe revert to `master` version (for this file only)?

Reply:
The unit tests for this use `com.beust.jcommander.internals` which was listed as an illegal import.   Not sure why that's the case, but I had to do that in order to make it work.

Reply:
Which aspect was being used? There is no comment in the config file to say why that library was excluded. Without that info, I'd kind of assume that there was a good reason, and find some other way to get the tests to work.

Reply:
That's definitely needs to be reverted.

Reply:
Reverted